### PR TITLE
minor tweaks

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

File: security/password_migration.rst [hunk 1]
```diff
@@ -150,7 +150,7 @@ for this login request. This password is used in the migration process::
 
         public function getPassword($credentials): ?string
         {
-            return $credentials['password'];
+            return $credentials['password'] ?? null;
```
Comment (wouterj) [inline security/password_migration.rst]: I'm not in favor of this change. It's a guard authenticator and credentials are equal to what's returned by `getCredentials()` of the same guard, so we know for sure which array items exists. If this is not the case, someone is misusing the guard interface :)

Comment (chalasr) [reply security/password_migration.rst]: I agree with @wouterj

Comment (javiereguiluz) [reply security/password_migration.rst]: I've made this change while merging. Thanks!

**Comment by javiereguiluz** [comment]
Thanks Christian!

**Pull request merged**